src/faker_generator.py

```diff
@@ -0,0 +1,60 @@
+
+import random
+from faker import Faker
+
+from src.models.Note import Note
+from src.models.NoteBook import NoteBook
+from src.models.Record import Record
+from src.models.AddressBook import AddressBook
+from src.constants import INFO
+
+fake = Faker()
+
+def generate_tags():
+    possible_tags = ["work", "personal", "urgent", "important", "low-priority", "ideas", "to-do"]
+    num_tags = random.randint(1, len(possible_tags))
+    return random.sample(possible_tags, num_tags)
+
+
```

Comment:
add @input_error decorator to handle errors in args parsing, also add str to int casting error in decorator if needed

Reply:
thanks! done